js/index.js

```diff
@@ -0,0 +1,142 @@
+
+var headlinesURL = "http://news-summary-api.herokuapp.com/guardian?apiRequestUrl=http://content.guardianapis.com/search?section=uk-news&from-date=2021-01-24&to-date=2021-01-24"
+
```

Comment:
Assigning the API link to a variable is a good idea , i will use this myself in the future